gtda/homology/grids.py

```diff
@@ -33,24 +33,35 @@ class CubicalPersistence(BaseEstimator, TransformerMixin):
         :math:`\\mathbb{F}_p = \\{ 0, \\ldots, p - 1 \\}` where
         :math:`p` equals `coeff`.
 
+    periodic_dimensions : tuple of bool, optional, default: ``None``
+        Tuple of `D` booleans, where `D` is the dimension of the images. The
+        boolean in the `d`th position expresses whether the boundaries along
+        the `d`th axis are periodic. ``None`` implies that none of the
+        boundaries are periodic.
+
     infinity_values : float or None, default : ``None``
         Which death value to assign to features which are still alive at
         filtration value `np.inf`. ``None`` assigns the maximum pixel
-        values within all images passed to meth:`fit`.
+        values within all images passed to :meth:`fit`.
 
     n_jobs : int or None, optional, default: ``None``
         The number of jobs to use for the computation. ``None`` means 1 unless
         in a :obj:`joblib.parallel_backend` context. ``-1`` means using all
         processors.
 
+    Attributes
+    ----------
+    infinity_values_ : float
+       Infinity value calculated in :meth:`fit`.
```

Comment:
...calculated in :meth:`fit`, or passed by the user?

Reply:
The attribute is set in `fit` based on the value of the parameter. I clarified it.